docs/cli/defradb_client_document.md

```diff
@@ -1,38 +0,0 @@
-## defradb client document
-
-Create, read, update, and delete documents.
```

Comment:
question: Why have these files been deleted?

Reply:
These seem outdated docs, do lemme know if anything isn't (my guess is that overtime the `make docs` command generates any new documents, but does not remove old ones).

Reply:
These are auto-generated?

Reply:
Yea so for example this was regenerated `docs/cli/defradb_client_collection_docIDs.md` with new file name but was originally `docs/cli/defradb_client_collection_keys.md`

And this file that I deleted: `defradb_client_document.md` I am pretty sure has been out dated since `defradb_client_collection.md`: https://github.com/shahzadlone/defradb/blob/gql-renaming-phase1/docs/cli/defradb_client_collection.md

Reply:
Ah okay nice - I did not know this :)

I've opened https://github.com/sourcenetwork/defradb/issues/2166 and https://github.com/sourcenetwork/defradb/issues/2165

Reply:
I didn't either LOL,

Cheers for opening these